en/docs/tutorials/create-your-first-scheduled-task.md

```diff
@@ -0,0 +1,218 @@
+
+First, let's create a new project and add a Scheduled Task component to it as follows:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
+
+2. Expand the drop-down menu for projects and click **+ Create New**.
+
+    ![Create project](../assets/img/tutorials/rest-api/create-project.png){.cInlineImage-full}
+
+    Enter the name and the description for the project as follows:
+
+    | **Field**       | **Value**                                             |
+    |-----------------|-------------------------------------------------------|
+    | **Name**        | `USPopulationStats`                                   |
+    | **Description** | `Routinely retrieves statistics on the US population` |
+
+    Then click **Create**.
+
+    This takes you to the **Components** page shown below.
+
+    ![Create component](../assets/img/tutorials/rest-api/create-component.png){.cInlineImage-full}
+
+3. Add a component as follows:
+
+    1. On the Components page, click **Create**, and then click **Scheduled Task**.
+
+    2. In the **Name** field, enter `us-population-stats` and click **Next**. The **Scheduled Task** dialog box opens.
+
+    3. For this tutorial, let's create the scheduled task configuration in a Choreo-managed repository. Therefore, leave the **Choreo-managed repository** selected (as by default) and click **Create**.
+       
+    As a result, this Scheduled Task component opens on a separate page.
+
+### Step 1.2: Design the scheduled task
+
+Let's design the functionality of the scheduled task by following this procedure:
+
+1. To design the process that the scheduled task needs to carry out, click **Edit Code**. As a result, the Web Editor in which you can develop the scheduled task in low-code or pro-code opens.
+
+    !!! info
+        Opening the Web Editor may take a little while if you are a first-time user.
+
+    The low-code diagram that you can see in the Web Editor is the default scheduled task configuration created by Choreo.
+
+2. To remove the construct of the default scheduled task configuration (which is of the `Main` type), hover over it until two icons appear for editing and deleting as shown in the image below.
+
+    ![Hover over main function](../assets/img/tutorials/scheduled-task/hover-over-main-function.png){.cInlineImage-half}
+
+    Once they appear, click the icon for deleting.
+
+3. Let's start creating the scheduled task by adding a construct of the `Main` type. To do this, click on the plus icon shown in the image below:
+
+    ![Add new function](../assets/img/tutorials/scheduled-task/add-new-function.png){.cInlineImage-half}
+
+    Click **Main**.
+
+    In the panel that appears on the right of the page, click **Save** to proceed with the default settings.
+
+4. To define the functionality of the scheduled task, you can update the low-code diagram that appears below the construct you added.
```

Comment:
As discussed over the call on 25th May 2022 4.30 PM, this approach is followed because editing the low-code diagram for the existing construct results in the "Invalid token" error.